src/cqlvalue_to_py.rs

```diff
@@ -0,0 +1,79 @@
+use pyo3::prelude::*;
+use pyo3::types::{PyBool, PyBytes, PyDict, PyFloat, PyInt, PyList, PyString};
+use scylla::value::CqlValue;
+
+pub fn cql_value_to_py(py: Python<'_>, v: &CqlValue) -> PyResult<PyObject> {
+    use CqlValue::*;
+
+    let obj = match v {
+        Ascii(s) | Text(s) => PyString::new(py, s).into_any(),
+
+        Boolean(b) => PyBool::new(py, *b).to_owned().into_any(),
+
+        Int(i) => PyInt::new(py, *i).into_any(),
+        BigInt(i) => PyInt::new(py, *i).into_any(),
+        SmallInt(i) => PyInt::new(py, *i).into_any(),
+        TinyInt(i) => PyInt::new(py, *i).into_any(),
+
+        Double(f) => PyFloat::new(py, *f).into_any(),
+        Float(f) => PyFloat::new(py, *f as f64).into_any(),
+
+        // Blob -> bytes
+        Blob(bytes) => PyBytes::new(py, bytes).into_any(),
+
+        // UUID
+        Uuid(u) => {
+            let uuid_mod = py.import("uuid")?;
+            let uuid_cls = uuid_mod.getattr("UUID")?;
+            uuid_cls.call1((u.to_string(),))?.into_any()
```

Comment:
How does Python imports work? Specifically, what is the performance of re-importing modules?
You will call this import and getattr for each deserialize, so if it is not optmized well in Python / pyo3, we may need to cache `uuid_mod` / `uuid_cls`, for example using https://doc.rust-lang.org/std/sync/struct.OnceLock.html